css/theme-weebly.scss

```diff
@@ -25,37 +25,37 @@ $sidebar-width: 220px;
 .nav-tabs > li.active > a,
 .nav-tabs > li.active > a:hover,
 .nav-tabs > li.active > a:focus {
-    background-color: #248ec2;
-    color: white !important;
+	background-color: #248ec2;
```

Comment:
Jekyll and this repo are using spaces. Mind reverting these style changes.

We should add a `.editorconfig` file to prevent these issues. Something like the following, though this repo uses uses indent_size = 4